app/src/main/kotlin/com/simplemobiletools/draw/pro/activities/MainActivity.kt

```diff
@@ -40,6 +40,7 @@ import java.io.ByteArrayOutputStream
 import java.io.File
 import java.io.OutputStream
 
+
```

Comment:
avoid adding such double empty lines please

Reply:
sorry, removed them. (was added automatically by the editor when formatting)